tests/integration/helpers.js

```diff
@@ -173,8 +173,6 @@ async function scan(name, scanType, parameters = [], timeout = 180) {
  * @returns {scan.findings} returns findings { categories, severities, count }
  */
 async function cascadingScan(name, scanType, parameters = [], { nameCascade, matchLabels }, timeout = 180) {
-  namespace = "cascading-tests";
-
```

Comment:
Pipeline is failing because I changed the namespace for cascading scans here. I did that to keep the Makefile for cascading scans easy, but I'll take a look again.

Reply:
Already looking into it :)
Changed it to run in the integration-tests namespace in the pipeline.
"Should be working now"™️

Currently only on this branch, will push to the pr when the pipeline passes: https://github.com/secureCodeBox/secureCodeBox/commits/makefile-scanners

Reply:
Seems to be working now.
Anything else from you point of view stopping this from getting merged?
Otherwise i think we can merge this asap